src/routes/analytics/new/NewFromTemplatePage.tsx

```diff
@@ -0,0 +1,40 @@
+import { Container, Title } from "@mantine/core"
+import {
+  CollectionBrowser,
+  ModifyQuestion,
+} from "@metabase/embedding-sdk-react"
+import { useAtom } from "jotai"
+import { selectedQuestionTemplateIdAtom } from "../../../store/create"
+import { QUESTION_TEMPLATE_COLLECTION_ID } from "../../../constants/collections"
+
+export const NewFromTemplatePage = () => {
+  const [templateQuestionId, setQuestionId] = useAtom(
+    selectedQuestionTemplateIdAtom,
+  )
+
+  if (templateQuestionId === null) {
+    return (
+      <Container>
+        <Title fz="28px" mb="md">
+          Pick a question
+        </Title>
+
+        <CollectionBrowser
+          collectionId={QUESTION_TEMPLATE_COLLECTION_ID}
+          visibleEntityTypes={["question"]}
+          onClick={(item) => setQuestionId(item.id)}
+        />
```

Comment:
Use the collection browser for selecting questions to use as templates, from the "Question Templates" collection.

![CleanShot 2567-09-04 at 02 18 30@2x](https://github.com/user-attachments/assets/971015ae-71d1-4f1e-a6b5-0b87a507a51c)